lib/article_json/import/google_doc/html/text_box_parser.rb

```diff
@@ -5,14 +5,16 @@ module HTML
         class TextBoxParser
           include Shared::Float
 
+          # @param [Nokogiri::HTML::Node] Current node
```

Comment:
It took a bit to understand why we pass a `current_node` in and what the difference of this and `nodes` is. Some explanation could help :) 

Also, I think as per our version of YARD what comes after `@param [...]` needs to be the parameter name. So this could be something like:
```suggestion
          # @param [Nokogiri::HTML::Node] current_node Document node that states
                                                       that this is a textbox. 
                                                       May contain tags, too.
```

Reply:
Another idea might be to even rename the param to something like `type_node` or `first_node`...